wgsl/index.bs

```diff
@@ -7848,13 +8037,17 @@ the fragment will be thrown away.
 <div class='syntax' noexport='true'>
   <dfn for=syntax>func_call_statement</dfn> :
 
-    | [=syntax/ident=] [=syntax/argument_expression_list=]
+    | [=syntax/attribute=] * [=syntax/call_phrase=]
 </div>
 
 A function call statement executes a [=function call=].
 
 Note: If the function [=return value|returns a value=], that value is ignored.
 
+Note: The only attribute that can be applied is the [=attribute/diagnostic=] attribute,
+and only when calling a
+[=builtin functions that compute a derivative|builtin function that computes a derivative=].
```

Comment:
This seems to go too far. If there are non-standard diagnostic rule names they can only be applied to derivatives?

Reply:
Wil remove the note.